addons/hr/__manifest__.py

```diff
@@ -19,6 +19,7 @@
         'mail',
         'resource',
         'web',
+        'base_install_request',
```

Comment:
The point of having this feature in its own module was that it could be uninstalled for customers who don't want it.

Having it as a dependency of other modules defeats this purpose.
And this is clearly not a dependency per se.

Reply:
Do you prefer a post_init_hook in ``base_install_request`` to force the modules installation (knowledge, etc) ?

Reply:
One advantage of using a init hook rather than the regular auto-install mechanism would be to make it controllable. It could support a special environment parameter and/or a server config parameter to disable the auto-installing.

This would be useful for developers and administrators who have a good reason to disable it, without changing the result for everyone else?

Reply:
to clarify, I don't mean an ICP, but a server.conf entry that developers can use to speed up dev/testing (not exposed in the CLI parameters), e.g. `tools.config["default_productivity_apps"] in (True, False)`
and/or an `os.environment` variable e.g. `ODOO_NO_DEFAULT_PRODUCTIVITY_APPS` or something, to be used during upgrades to skip the auto-installation